src/Sylius/Component/Core/Model/Image.php

```diff
@@ -118,6 +117,38 @@ public function setPath($path)
     /**
      * {@inheritdoc}
      */
+    public function getTitle()
```

Comment:
Other classes rely on the ImageInterface and not all need a title and description
The Taxon model is a example:

    class Taxon extends BaseTaxon implements ImageInterface, TaxonInterface